.github/workflows/minimal.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: psr -s create-container-image -c ${{ inputs.stepRunnerConfigDir }}/psr.yaml /ploigos/generated-psr
 
       - name: ci-push-container-image-to-repository
-        run: psr -s push-container-image -c ${{ inputs.stepRunnerConfigDir }}/psr.yaml /ploigos/generated-psr --step-config container-image-push-repository=aagreen/java-github-example
+        run: psr -s push-container-image -c ${{ inputs.stepRunnerConfigDir }}/psr.yaml /ploigos/generated-psr
```

Comment:
Doesn't have to be in this PSR, but can we please change "generated-psr" to "shared-psr.yaml"? I want the file extension to make it clear it's an analogue to psr.yaml, and shared to express that multiple apps will use it.